backend/sozisel/lib/sozisel/model/event_results/event_result.ex

```diff
@@ -23,7 +24,11 @@ defmodule Sozisel.Model.EventResults.EventResult do
     field :result_data, PolymorphicEmbed,
       types: [
         quiz_result: [module: QuizResult, identify_by_fields: [:participant_answers]],
-        poll_result: [module: PollResult, identify_by_fields: [:option_ids]]
+        poll_result: [module: PollResult, identify_by_fields: [:option_ids]],
+        whiteboard_result: [
+          module: WhiteboardResult,
+          identify_by_fields: [:metadata, :path, :text, :used_time]
```

Comment:
We don't use the metadata field even for recordings, unless you have some special usecase for it just delete the field.

Reply:
Okay, I will remove this field both for photos and for recordings